app/src/MainController.cpp

```diff
@@ -0,0 +1,290 @@
+    auto camera   = graphics->camera();
+    graphics->begin_gui();
+    // Draw Camera Info window
+    {
+        ImGui::Begin("Camera info");
+        ImGui::SetNextWindowPos(ImVec2(500, 350));
+        ImGui::SetNextWindowSize(ImVec2(710, 300));
+        const auto &c = *camera;
+        ImGui::Text("Camera position: (%f, %f, %f)", c.Position.x, c.Position.y, c.Position.z);
+        ImGui::Text("(Yaw, Pitch): (%f, %f)", c.Yaw, c.Pitch);
+        ImGui::Text("Camera front: (%f, %f, %f)", c.Front.x, c.Front.y, c.Front.z);
+        ImGui::SliderFloat("Camera x", &camera->Position.x, -20.f, 20.0f, "%.2f");
+        ImGui::SliderFloat("Camera y", &camera->Position.y, -20.f, 20.0f, "%.2f");
+        ImGui::SliderFloat("Camera z", &camera->Position.z, -20.f, 20.0f, "%.2f");
+
+        ImGui::Text("Point light sliders:");
+        ImGui::Text("Point light ambient intensity: (%f, %f, %f)", this->light.ambient.x, this->light.ambient.y,
+                    this->light.ambient.z);
+        ImGui::SliderFloat3("Point light ambient slider", glm::value_ptr(this->light.ambient), 0.1f, 1.f);
+        ImGui::Text("Point diffuse light intensity: (%f, %f, %f)", this->light.diffuse.x, this->light.diffuse.y,
+                    this->light.diffuse.z);
+        ImGui::SliderFloat3("Point diffuse slider", glm::value_ptr(this->light.diffuse), 0.1f, 1.f);
+        ImGui::Text("Point light specular intensity: (%f, %f, %f)", this->light.specular.x, this->light.specular.y,
+                    this->light.specular.z);
+        ImGui::SliderFloat3("Point specular slider", glm::value_ptr(this->light.specular), 0.1f, 1.f);
+
+        ImGui::Text("Spot light sliders:");
+        ImGui::Text("Spot light diffuse intensity: (%f, %f, %f)", this->spotLight.diffuse.x, this->spotLight.diffuse.y,
+            this->spotLight.diffuse.z);
+        ImGui::SliderFloat3("Spot light diffuse slider", glm::value_ptr(this->spotLight.diffuse), 0.1f, 1.f);
+        ImGui::Text("Spot light diffuse intensity: (%f, %f, %f)", this->spotLight.specular.x, this->spotLight.specular.y,
+            this->spotLight.specular.z);
+        ImGui::SliderFloat3("Spot light specular slider", glm::value_ptr(this->spotLight.specular), 0.1f, 1.f);
+        ImGui::Text("Cut off angle: (%f)", this->spotLight.cut_off);
+        ImGui::SliderFloat("Cut off slider", &this->spotLight.cut_off, -1.0f, 1.f);
+        ImGui::Text("Outer cut off angle: (%f)", this->spotLight.outterCut_off);
+        ImGui::SliderFloat("Outer cut off slider", &this->spotLight.outterCut_off, -1.0f, 1.f);
+        ImGui::End();
+    }
+    graphics->end_gui();
+}
+
+void MainController::draw() {
+    draw_framebuffer_rectangle();
+    draw_moon();
+    draw_space_station();
+    draw_space_craft();
+    draw_skybox();
+    auto graphics = engine::core::Controller::get<engine::graphics::GraphicsController>();
+    auto platform = engine::core::Controller::get<engine::platform::PlatformController>();
+    auto camera   = graphics->camera();
```

Comment:
Izgleda da graphics i camera nisu korišćeni u ovoj funkciji. Vredno je voditi računa o održavanju koda urednim i konzistentnim, jer se s vremenom ovakve stvari mogu nagomilati, što otežava kasnije refaktorisanje i razumevanje koda.